### DISCUSSION TITLE
#44 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by Riely007** [tog-ETH-er/togETHer#21] [General]
Agenda:

1. 下一届轮值主席提名投票
2. Community updates
3. Fireside discussion

> 会议主题：togETHer
> 会议时间： 21:00-22:00
> 重复周期：每两周 (周一)
> 
> 点击链接入会，或添加至会议列表：
> https://meeting.tencent.com/s/LxNajKWJw4RV
> 
> 会议 ID：891 5098 6775

**Comment by bambalados** [comment]
近两周举办了1场线上活动:

- CTN Let's Talk3 "[ 借你一双 #GenZ 的眼睛👀- 从GenZ视角看 #Web3](https://twitter.com/i/spaces/1gqxvlyVkjjGB)" (Twitter Space)
- 正在计划 CTN#16, 目前计划是DappLearning的小伙伴介绍Euler Protocol，和Evabase的小伙伴做Workshop.

**Comment by liushooter** [comment]
## Rebase

### workshop

举办了 2 场线上 workshop

- [链上安全角度分析 Luna/Celsius/3AC 暴雷问题](https://www.bilibili.com/video/BV1hB4y1a7dC)  By SharkTeam

- [区块链与大数据最佳实践](https://github.com/rebase-network/event-pool/issues/47) By jason

### 日常
- [Web3 日报](https://rebase.network/posts/category/web3daily) 更新 14 期

- 更新多篇 [招聘](https://github.com/rebase-network/who-is-hiring)

- 更新多篇 [活动](https://github.com/rebase-network/event-pool)

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

8个PR #803 ～ #810

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了2场分享会

- 《解析我挖到的Ethermint链代币增发漏洞》 by iczc
  - https://www.youtube.com/watch?v=icK07IuX4po&t=85s

- 《不同市场下的期权策略》by Koi
  - https://www.youtube.com/watch?v=MDp7sMz__J0

## 翻译小组

- MIT公开课正式发布第20集
- 第21、22 审稿中

**Comment by stephanieeee-ho** [comment]
### ECN
1. 日常翻译和七日谈
2. 与普朗克社区一起举办 EIP-4337 的分享
https://www.bilibili.com/video/BV1j34y1J774?spm_id_from=333.999.0.0&vd_source=4f250c2896497165ff9e166ec652e32a

**Comment by Riely007** [comment]
### 以太坊翻译小组

1. 官网翻译进行中

**Comment by Emilyxiannv** [comment]
### DAOSquare
开始筹备孵化器上线的事情，在做一些marketing/AMA/PR的计划
